src/styles.scss

```diff
@@ -1,22 +1,29 @@
 /* You can add global styles to this file, and also import other style files */
 
-@import '~@angular/material/theming';
-@include mat-core();
-$app-primary: mat-palette($mat-indigo);
-$app-accent: mat-palette($mat-pink);
-$app-warn: mat-palette($mat-red);
+@use '@angular/material' as mat;
 
-// Custom Sass colors vars (will be available in all the project)
-$primary: mat-color($app-primary);
-$accent: mat-color($app-accent);
-$warn: mat-color($app-warn);
+@include mat.core();
 
-$theme: mat-light-theme($app-primary, $app-accent, $app-warn);
+$app-primary: mat.define-palette(mat.$indigo-palette);
+$app-accent: mat.define-palette(mat.$pink-palette);
+$app-warn: mat.define-palette(mat.$red-palette);
 
-@include angular-material-theme($theme);
+$theme: mat.define-light-theme(
+  (
+    color: (
+      primary: $app-primary,
+      accent: $app-accent,
+      warn: $app-warn,
+    ),
+    typography: mat.define-typography-config(),
+    density: 0,
+  )
+);
+
+@include mat.all-component-themes($theme);
 
 // Use this import in a real project instead
-//@import '~@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
+//@import '@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
```

Comment:
sorry, I know this is definitely something I should do, but I do not find any time. would you mind updating the [theming part of the `README.md`](https://github.com/kuhnroyal/mat-datetimepicker#theming) as well?